id/prefix.go

```diff
@@ -0,0 +1,33 @@
+package id
+
+type Prefix interface {
+	GetLength() int
+	isPrefixOf(Id) bool
```

Comment:
Is there a reason this method is private?

Reply:
No, thank you for the catch!